src/browser/api/window.js

```diff
@@ -477,7 +477,7 @@ Window.create = function(id, opts) {
             });
 
             // can't unhook when the 'closed' event fires; browserWindow is already destroyed then
-            browserWindow.webContents.removeAllListeners('page-favicon-updated');
+            webContents.removeAllListeners('page-favicon-updated');
```

Comment:
@rdepena no i did not; not worth the effort to test it; I'll just change it back. Point taken!